src/openmrs-fetch.test.ts

```diff
@@ -175,6 +184,34 @@ describe("openmrsFetch", () => {
         expect(err.response.status).toBe(400);
       });
   });
+
+  it(`redirects to login page when the server responds with a 401`, done => {
```

Comment:
You don't need the `done` function when returning a promise with jest. See https://jestjs.io/docs/en/asynchronous#promises

```suggestion
  it(`redirects to login page when the server responds with a 401`, () => {
```